backends/gdx-backends-gwt/src/com/badlogic/gwtref/client/ReflectionCache.java

```diff
@@ -44,35 +44,47 @@ private static String convert (String className) {
 		if (className.startsWith("[")) {
 			int dimensions = 0;
 			char c = className.charAt(0);
-			String suffix = "";
+			StringBuilder stringBuilder = new StringBuilder();
 			while (c == '[') {
 				dimensions++;
-				suffix += "[]";
 				c = className.charAt(dimensions);
 			}
 			char t = className.charAt(dimensions);
 			switch (t) {
 			case 'Z':
-				return "boolean" + suffix;
+				stringBuilder.append("boolean");
+				break;
 			case 'B':
-				return "byte" + suffix;
+				stringBuilder.append("byte");
+                break;
 			case 'C':
-				return "char" + suffix;
+                stringBuilder.append("char");

```

Comment:
Please indent all lines in all files with tabs

Reply:
[fix](https://github.com/libgdx/libgdx/pull/5670/commits/2fb6972b4fa3746cf37e5e181f0e9e10680f27e2)